witness/src/Dscp/Witness/Util.hs

```diff
@@ -0,0 +1,9 @@
+
+module Dscp.Witness.Util (dieGracefully) where
```

Comment:
Please move to `Core.Util`, this single function does not require any specific witness modules. Also, `Util` is a mess and we'll figure it out at once later, let's keep the garbage compact ;)

Reply:
Will do.